engine/src/test/scala/cromwell/webservice/MetadataBuilderActorSpec.scala

```diff
@@ -292,15 +291,16 @@ class MetadataBuilderActorSpec extends TestKitSuite("Metadata") with FlatSpecLik
     val kisv3 = ("key[0]:subkey", "value3", OffsetDateTime.now.plusSeconds(2))
     val kiv4 = ("key[0]", "value4", OffsetDateTime.now.plusSeconds(3))
 
-    val t = Table(
-      ("list", "res"),
+    val t = List(
       (List(kv),  """"key": "value""""),
       (List(kv, ksv2),  """"key": { "subkey": "value2" }"""),
       (List(kv, ksv2, kisv3),  """"key": [ { "subkey": "value3" } ]"""),
       (List(kv, ksv2, kisv3, kiv4),  """"key": [ "value4" ]""")
     )
 
-    forAll(t) { (l, r) => assertMetadataKeyStructure(l, r) }
+    Future.sequence(t map { case (l, r) => assertMetadataKeyStructure(l, r) }) map { assertions =>
```

Comment:
I'm happy to do this if you want me to but that doesn't really seem worth it, particularly for test code. OTOH there's more 😼 so that's something. :) I'll leave it up to you.